.github/workflows/updatecli.yml

```
@@ -0,0 +1,28 @@
+name: updatecli
+
+on:
+  workflow_dispatch: ~
+  schedule:
+    - cron: '0 6 * * *'
+
+permissions:
+  contents: write
```

Is using the standard GH actions bot with these permissions okay here?

---

This won't trigger a build. In this particular case we use

```suggestion
  contents: read
```


And a new secret called `UPDATECLI_GH_TOKEN`. I'll need to create the secret